sqlalchemy.exc.ResourceClosedError: This result object does not return rows. It has been closed automatically.